transformer_lens/loading_from_pretrained.py

```diff
@@ -758,6 +798,28 @@ def convert_hf_model_config(model_name: str, **kwargs):
             "scale_attn_by_inverse_layer_idx": hf_config.scale_attn_by_inverse_layer_idx,
```

Comment:
Can you add trust_remote_code: True if model_name contains santacoder else False or something here?

Reply:
done